### ISSUE TITLE
Luminosity contrast ratio of keyboard focus indicator on interactive controls present in the "Demo conference API screen" fails to meet the required ratio of 3:1.: A11y_VS Code extension for API Center_Definition_OpenAPIDocumentation_NonTextContrast

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/vscode-azureapicenter#110]
### GitHub Tags
#A11yTCS; #A11yMAS; #A11ySev2; #WCAG1.4.11; #GH_VSCodeextensionforAPICenter_Linux_Apr2024; #DesktopApp; #VS Code Extension for API Center; #Non-TextContrast; #Linux; #FTP; #A11yWCAG2.1;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Linux- Ubuntu 22.04.4

### Repro Steps:

1. Create an API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the Api name which you have created, and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. Go to "Open API Documentation" using the down arrow and press enter. 
7. Demo conference API screen will appear.
8. Verify luminosity contrast ratio of keyboard focus indicator on interactive controls present in the "Demo conference API screen" fails to meet the required ratio of 3:1.

### Actual Result:
Luminosity contrast ratio of keyboard focus indicator on interactive controls present in the "Demo conference API screen" fails to meet the required ratio of 3:1.

### Expected Result:
Luminosity contrast ratio of keyboard focus indicator on interactive controls present in the "Demo conference API screen" should be greater than or equal to required ratio of 3:1.

### User Impact:
People with low vision often have difficulty perceiving tab focus that have insufficient contrast if Luminosity contrast ratio of visual focus indicator fails to meet the required ratio of 3:1. This can be exacerbated if the person has a color vision deficiency that lowers the contrast even further. Providing a lightness difference of 3:1 or greater can make these items more distinguishable when the person does not see a full range of colors.

**Comment by AnKushSingh05** [comment]
IRT Bug logged: https://github.com/microsoft/vscode-azureapicenter/issues/112

**Comment by Yash14j** [comment]
### GitHub Tags:
#External;#External:SwaggerUI;#eDAD:3p:222939;#ThirdParty;

**Comment by AnKushSingh05** [comment]
External bug: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/222939

**Comment by formulahendry** [comment]
vs code bug: https://github.com/microsoft/vscode/issues/237882

**Comment by Yash14j** [comment]
GithubTags:#ResolvedByTT;

**Comment by keerthiduvvuri** [comment]
GithubTags:#ResolvedByTT
